deploy/deployment/openshift/objects/devworkspace-controller-manager.Deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - name: WEBHOOK_SECRET_NAME
           value: devworkspace-webhookserver-tls
         - name: RELATED_IMAGE_devworkspace_webhook_server
-          value: quay.io/devfile/devworkspace-controller:next
+          value: quay.io/amisevsk/devworkspace-controller:dev
```

Comment:
Seems it needs to be reverted

Reply:
My bad, I've started setting `DEFAULT_IMAGE` to use my development image in the OLM deployment as well.